core/src/test/java/org/frankframework/jdbc/MessageStoreListenerTest.java

```diff
@@ -36,7 +34,7 @@ public class MessageStoreListenerTest {
 
 	@BeforeEach
 	public void setup(DatabaseTestEnvironment env) throws Exception {
-		assumeThat(env.getDbmsSupport().getDbms(),equalTo(Dbms.H2)); // tests are based on H2 syntax queries
+		assumeTrue(Dbms.H2 == env.getDbmsSupport().getDbms()); // tests are based on H2 syntax queries
```

Comment:
So the tests are only executed for H2 and for all other databases they are skipped? Is that right actually, that we will never test against other databases when they're running?